src/Build/Globbing/MSBuildGlob.cs

```diff
@@ -117,23 +117,20 @@ public MatchInfoResult MatchInfo(string stringToMatch)
         {
             ErrorUtilities.VerifyThrowArgumentNull(stringToMatch, nameof(stringToMatch));
 
-            if (FileUtilities.PathIsInvalid(stringToMatch) ||
-                !IsLegal)
+            if (FileUtilities.PathIsInvalid(stringToMatch) || !IsLegal)
             {
                 return MatchInfoResult.Empty;
             }
 
             var normalizedInput = NormalizeMatchInput(stringToMatch);
```

Comment:
not a change in the PR.  It's difficult to infer normalizedInput's type.

Reply:
Fixed.